loginsystem/newsfeed/images/News Feed.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<!-- saved from url=(0069)file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/splashpage.html -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <link rel="stylesheet" type="text/css" href="file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/CSS/style.css?version=1">
+
+  <title>News Feed</title>
+
+</head>
+<body>
+
+<div class="grid-container">
```

Comment:
name isn't descriptive enough. grid-container doesn't mean much in the context of the whole file. pick a better name that is more representative of the container you are creating. something relatable to the blogContainer.. or anything of the like.